ultrasonic/src/main/kotlin/org/moire/ultrasonic/util/Util.kt

```diff
@@ -1097,6 +1097,13 @@ object Util {
         return preferences.getBoolean(Constants.PREFERENCES_KEY_SEND_BLUETOOTH_ALBUM_ART, true)
     }
 
+    fun getShouldDisableNowPlayingListSending(): Boolean {
+        val preferences = getPreferences()
+        return preferences.getBoolean(
+            Constants.PREFERENCES_KEY_DISABLE_SEND_NOW_PLAYING_LIST, false
+        )
+    }
+
     @JvmStatic
     fun getViewRefreshInterval(): Int {
```

Comment:
Same here. There is a nice refactor tool in Android Studio, in the context menu, called "convert to property" or something like this.